Java: Query to detect Android backup allowed (#10106)
github/codeql: java/ql/lib/change-notes/2022-08-18-android-manifest-backup-predicate.md

---
category: feature
---

> * Added a new predicate, `allowsBackup`, in the `AndroidApplicationXmlElement` class. This predicate detects if the application element has its `android:allowBackup` attribute enabled.

Reviewer: This needs adjustment after the change in `allowsBackup`:

```suggestion
* Added a new predicate, `allowsBackup`, in the `AndroidApplicationXmlElement` class. This predicate detects if the application element does not disable the `android:allowBackup` attribute.
```
